lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -1159,6 +1159,14 @@ static std::string GetPluginServerForSDK(llvm::StringRef sdk_path) {
   return server_or_err->str();
 }
 
+namespace {
+  static constexpr std::array<std::string_view, 4> knownExplicitModulePrefixes =
+       {"-fmodule-map-file=",
+        "-fmodule-file=",
+        "-fno-implicit-modules",
+        "-fno-implicit-module-maps"};
+}
```

Comment:
FWIW `static` and anonymous namespaces are the same thing: both have internal linkage and static storage duration